metomi/rosie/graph.py

```diff
@@ -22,31 +22,33 @@
 import textwrap
 import time
 
-import rose.metadata_graph
-import rose.opt_parse
-import rose.reporter
-import rosie.suite_id
-import rosie.ws_client
-import rosie.ws_client_cli
+import pygraphviz
```

Comment:
This is raising an issue in my virtual environment, due to `pygraphviz` not being installed after `pip install .`. I don't see any graphviz related dependencies in `setup.py`... is this file used by `rose` or `rosie`? If so, we may need to include the dependencies in `setup.py`.

Reply:
Yes. Thanks! Will double check here if we want to kill this dependency, but if not it needs adding.

Reply:
Yep, this dependency has been left floating about, it will be removed in due course, I think we will copy the solution from the new Cylc Graph.

Reply:
For now either delete `rosie graph` or add `pygraphviz` to the dependencies (note it's a bit of a pain to install).

Reply:
@kinow I'm afraid I'm just going to leave this for this PR and create a new issue to deal with removing this. There's no point getting this test working as the item tested will be removed and re-written in the new GUI. I do not want to include pygraphviz in the package dependencies since the need for it will be removed.